keyboards/keebio/levinson/keymaps/dcompact/keymap.c

```diff
@@ -0,0 +1,239 @@
+#include QMK_KEYBOARD_H
+
+#include "action_layer.h"
```

Comment:
`#include QMK_KEYBOARD_H` includes this files automatically, so Line 3 is redundant.

```suggestion
```

Reply:
Got it -- thank you for the suggestion!